lib/EntryManager/tests/entry-manager-test.js

```diff
@@ -15,131 +16,223 @@ import {
-                await entryManagerInteractor.detailsSection.actionsDropdown.edit.click();
+                await entryManagerInteractor.detailsSection.editItemButton.click();
               });
 
               it('entry form should be presented', () => {
                 expect(entryManagerInteractor.entryForm.isPresent).to.be.true;
               });
             });
 
-            describe('item duplication', () => {
+            describe('actions dropdown', () => {
+              describe('item edit', () => {
+                beforeEach(async () => {
+                  await entryManagerInteractor.detailsSection.actionsButton.click();
+                  await entryManagerInteractor.detailsSection.actionsDropdown.whenDeleteLoaded();
+                  await entryManagerInteractor.detailsSection.actionsDropdown.edit.click();
+                });
+
+                it('entry form should be presented', () => {
+                  expect(entryManagerInteractor.entryForm.isPresent).to.be.true;
+                });
+              });
+
+              describe('item duplication', () => {
+                beforeEach(async () => {
+                  await entryManagerInteractor.detailsSection.actionsButton.click();
+                  await entryManagerInteractor.detailsSection.actionsDropdown.whenDeleteLoaded();
+                  await entryManagerInteractor.detailsSection.actionsDropdown.duplicate.click();
+                });
+
+                it('entry form should be presented', () => {
+                  expect(entryManagerInteractor.entryForm.isPresent).to.be.true;
+                });
+              });
+            });
+
+            describe('item deletion prohibit modal', () => {
               beforeEach(async () => {
                 await entryManagerInteractor.detailsSection.actionsButton.click();
                 await entryManagerInteractor.detailsSection.actionsDropdown.whenDeleteLoaded();
-                await entryManagerInteractor.detailsSection.actionsDropdown.duplicate.click();
+                await entryManagerInteractor.detailsSection.actionsDropdown.delete.click();
               });
 
-              it('entry form should be presented', () => {
-                expect(entryManagerInteractor.entryForm.isPresent).to.be.true;
+              it('should not be presented', () => {
+                expect(entryManagerInteractor.isProhibitDeleteModalOpen).to.be.false;
               });
             });
           });
+        });
+      });
+    });
+  });
 
-          describe('item deletion prohibit modal', () => {
-            beforeEach(async () => {
-              await entryManagerInteractor.detailsSection.actionsButton.click();
-              await entryManagerInteractor.detailsSection.actionsDropdown.whenDeleteLoaded();
-              await entryManagerInteractor.detailsSection.actionsDropdown.delete.click();
-            });
+  describe('Test 422 error', () => {
+    const resp422 = new Response();
```

Comment:
I am wondering if it is possible to do like this to simplify things

```
     new Response(422, { 'Content-Type': 'application/json' }, JSON.stringify({
        errors: [{
          message: 'Cannot create entity; name is not unique',
          code: 'name.duplicate'
        }]
      }));
```

Reply:
Yes, you are right, because of json method, this notation is not available